docs/components/table/README.md

```diff
@@ -147,6 +148,7 @@ Supported field properties:
 | `thClass` | String or Array | Class name (or array of class names) to add to header/footer `<th>` cell
 | `tdClass` | String or Array | Class name (or array of class names) to add to data `<td>` cells in the column
 | `thStyle` | Object | JavaScript object representing CSS styles you would like to apply to the table field `<th>`
+| `formatter` | String or Function | A formatter callback function, can be used instead of slots 
```

Comment:
Yep, and by design these featues, slots and formatter are mutually exclusive. Ie. if we define formatter at items array, then we cant use slots, and slots are used **only** if formatter is not defined.